src/dataframe/dataframe.jl

```diff
@@ -589,10 +580,8 @@ end
 ##############################################################################
 
 """
-    insertcols!(df::DataFrame, ind::Int; name=col,
-                makeunique::Bool=false)
-    insertcols!(df::DataFrame, ind::Int, (:name => col)::Pair{Symbol,<:AbstractVector};
-                makeunique::Bool=false)
+    insertcols!(df::DataFrame, ind::Int, (name=>col)::Pair{Symbol,<:AbstractVector}...;
+                makeunique::Bool=false, copycols::Bool=false)
```

Comment:
Maybe we should use `copycols=true` by default? Not sure.

Reply:
I have thought about it. I rejected it because it would be breaking. But given that `insertcols!` is not used frequently maybe let us do it. It is similar case as `DataFrame` constructor with keyword arguments. We copy there by default. I will make the change.